extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -134,7 +146,10 @@ class PaginationButton {
     this.element.classList.remove(this.state_.className);
     this.element.classList.add(state.className);
     state.label
-      ? this.buttonElement_.setAttribute('aria-label', state.label)
+      ? this.buttonElement_.setAttribute(
+          'aria-label',
+          this.localizationService_.getLocalizedString(state.label)
```

Comment:
Can you get this string directly from the `buttonElement_` instead of getting it a second time?

Reply:
I don't think so. The button could need an updated string when updating the state.. e.g. before the last page of the story the button would contain the string "next page", but after navigating and running `updateState()` the string would need to be updated to "open bookend".

Reply:
SG thank you, I misread the code I thought it was building the templates :))